http-gateway/serverMux/jsonMarshaler.go

```diff
@@ -135,13 +135,13 @@ func modify(v interface{}) (newValue interface{}, wantDelete bool) {
 func (j *JsonMarshaler) Marshal(v interface{}) ([]byte, error) {
 	data, err := j.JSONPb.Marshal(v)
 	if err != nil {
-		return data, err
+		return nil, err
 	}
 
 	var val interface{}
 	err = json.Decode(data, &val)
 	if err != nil {
-		return data, nil
+		return nil, err
```

Comment:
return `data, nil`

Reply:
Only this line? What about line 157 I think we talked yesterday about that as well.

Reply:
Also, I'd like to add comment to the code. But ...I forgot it. :D Could you please formulate a short comment about why we return data in this case?